redCOMPONENT-COM/redSHOP
# XTREM-56 #comment Adding new tags to show transaction id

https://redweb.atlassian.net/browse/XTREM-56
# Testing Info:
- Place an order with any 3rd party payment gateway which gives payment transaction id 
- Add `{transaction_id}` and `{transaction_id_label}` tags in **Order Detail, Order Receipt, Order Mail** template and check.
